src/com/mambu/apisdk/MambuAPIServiceFactory.java

```diff
@@ -51,8 +52,32 @@ private MambuAPIServiceFactory(Injector injector) {
 	 * 
 	 * @return factory object to create API service objects which are bound to the given credentials
 	 */
+	@Deprecated
```

Comment:
The same comment as above, we should remove @Deprecated and change to "Convenience" method